### PR TITLE
docs: Document VAPID key should be base64 encoded

### PR DESCRIPTION
**Pull request by tarikeshaq**
Realized there was no documentation on the format of the VAPID key, read the source code and realized we call https://github.com/mozilla-services/autopush-rs/blob/6624a19ad97af90174cc7623b9b8683392c3aec9/autopush-common/src/util/mod.rs#L30-L32

(for reference, here's where we initiate the decoding while [making the endpoint](https://github.com/mozilla-services/autopush-rs/blob/6624a19ad97af90174cc7623b9b8683392c3aec9/autopush-common/src/endpoint.rs#L26)

cc @jrconlin

### CODE REVIEW

**Review by jonalmeida — APPROVED** [review]
Nice!

As mentioned offline, if we want to add some simple checks to avoid common mistakes, we could do something similar to [Gecko's WebPushSubscription construction](https://searchfox.org/mozilla-central/rev/78a2c17cc80680a5a82446e4ce7c45a73b935383/mobile/android/geckoview/src/main/java/org/mozilla/geckoview/WebPushSubscription.java#96-114).

File: docs/src/http.md [hunk 1]
```diff
@@ -249,6 +249,14 @@ An example of the Authorization header would be:
 ```html
     Authorization: Bearer 00secret00
 ```
+**{vapidKey}**  
+_The [VAPID Key](https://datatracker.ietf.org/doc/html/draft-ietf-webpush-vapid-01) provided by the subscribing third party_
```
Comment (jrconlin) [inline docs/src/http.md:253]: It's a real RFC, no need to refer to the draft (particularly a very old draft).

```suggestion
_The [VAPID Key](https://datatracker.ietf.org/doc/html/rfc8292#section-3.2) provided by the subscribing third party_
```

**Review by jrconlin — CHANGES_REQUESTED** [review]
Really, just the one change to use the proper RFC.

**Review by jrconlin — APPROVED** [review]

**Comment by tarikeshaq** [comment]
Hmm I've seen this sometimes in the past, lemme try closing and re-opening the PR

**Comment by jrconlin** [comment]
Rather than fight CI, https://github.com/mozilla-services/autopush-rs/pull/535

**Pull request closed without merging**